akka-actor/src/main/scala/akka/japi/JavaAPI.scala

```diff
@@ -60,7 +60,7 @@ trait Predicate[T] {
  * Additional tuple types for 3 to 22 values are defined in the `akka.japi.tuple` package, e.g. [[akka.japi.tuple.Tuple3]].
  */
 @SerialVersionUID(1L)
-case class Pair[A, B](first: A, second: B) {
+case class Pair[+A, +B](first: A, second: B) {
```

Comment:
javadsl.Flow.monitor wouldn't compile otherwise - Out is covariant in Flow/FlowOps but Pair isn't, so I can't return Pair<Out, FlowMonitor>